apps/studio/src/schemas/folder.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod'
+
+export const createFolderSchema = z.object({
+  folderTitle: z.string(),
```

Comment:
do we have restrictions on the length of the fields?

Reply:
we should! will confirm and update the fields

Reply:
Added in https://github.com/opengovsg/isomer/pull/149/commits/a915651965c55f32283ea5199230f5057d142d7a